### ISSUE TITLE
Prevent remove _thumbnail_id in data migration

### ISSUE THREAD

**Issue by claudiosanches** [woocommerce/woocommerce-product-tables-feature-plugin#153]
We should preserve `_thumbnail_id` in `wp_postmeta` for WordPress use, currently we are cleaning it while migrating data, what causes to have an empty image on admin.

**Comment by svaldesm** [comment]
Hi @claudiosanches is this project still alive?

**Comment by claudiosanches** [comment]
@svaldesm yes.

**Comment by svaldesm** [comment]
@claudiosanches awesome, thanks. I just realized this is an official repo from WC. Will have an eye on this.

**Comment by Jorgu5** [comment]
Yeah, the thumbnails are gone. I've migrated the products but no images are showing up on admin nor on the front-end site.
